scipy/ndimage/interpolation.py

```diff
@@ -32,17 +32,24 @@
 
 import math
 import numpy
+import warnings
+
 from . import _ni_support
 from . import _nd_image
-from . import _ni_docstrings
+from ._ni_docstrings import docdict
+from scipy.misc import doccer
 
-import warnings
+# Change the default 'reflect' to 'constant'
+docdict['mode'] = docdict['mode'].replace("Default is 'reflect'",
+                                          "Default is 'constant'")
```

Comment:
It looks to me like this needs `docdict = docdict.copy()` above it, to prevent altering the original dict. The rendered docstrings look correct, but that may depend on import order.

Reply:
Aha, I didn't even think about that, indeed let's stay on the safe side.